### ISSUE TITLE
[0.6.0 rc1] Crashes when running the event loop due to uninitialized memory on rust 1.48+

### ISSUE THREAD

**Issue by junkmail22** [ggez/ggez#898]
When running the event loop, ggez immediately crashes due to uninitialized memory. This occurs in 5.x as well.

Steps to reproduce: attempted to create a minimal example to reproduce, failed. I only get this issue in my monstrously complicated current game project, and not in any minimal reproduced examples.
```
Full runtime error backtrace:

thread 'main' panicked at 'attempted to leave type `linked_hash_map::Node<u32, glyph_brush_draw_cache::Row>` uninitialized, which is invalid', /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\core\src\mem\mod.rs:659:9

stack backtrace:

   0:     0x7ff6701a99be - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:108

   1:     0x7ff6701a99be - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66

   2:     0x7ff6701a99be - std::sys_common::backtrace::_print_fmt
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:67

   3:     0x7ff6701a99be - std::sys_common::backtrace::_print::{{impl}}::fmt
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:46

   4:     0x7ff6701c2e2b - core::fmt::write
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\core\src\fmt\mod.rs:1078

   5:     0x7ff6701a4118 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\io\mod.rs:1518

   6:     0x7ff6701ac7ad - std::sys_common::backtrace::_print
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:49

   7:     0x7ff6701ac7ad - std::sys_common::backtrace::print
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:36

   8:     0x7ff6701ac7ad - std::panicking::default_hook::{{closure}}
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:208

   9:     0x7ff6701ac388 - std::panicking::default_hook
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:227

  10:     0x7ff6701ad11f - std::panicking::rust_panic_with_hook
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:593

  11:     0x7ff6701acbf3 - std::panicking::begin_panic_handler::{{closure}}
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:497

  12:     0x7ff6701aa2ff - std::sys_common::backtrace::__rust_end_short_backtrace<closure-0,!>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:141

  13:     0x7ff6701acb79 - std::panicking::begin_panic_handler
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:495

  14:     0x7ff6701c0900 - core::panicking::panic_fmt
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\core\src\panicking.rs:92

  15:     0x7ff6701c084c - core::panicking::panic
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\core\src\panicking.rs:50

  16:     0x7ff66ffb49ea - linked_hash_map::LinkedHashMap<K,V,S>::insert::heb645e5327d76e61

  17:     0x7ff67001a95f - glyph_brush_draw_cache::DrawCache::cache_queued::h813f93459e4796ca

  18:     0x7ff66ffcef59 - glyph_brush::glyph_brush::GlyphBrush<V,X,F,H>::process_queued::h0470c49e4a022d9e

  19:     0x7ff66ffdceba - <ggez::graphics::text::Text as ggez::graphics::Drawable>::draw::hdab575d9d208894b

  20:     0x7ff66ff9201a - wars_prototype::menu_util::draw_list::h09d8e4ae07fc6aae

  21:     0x7ff66ff4daa1 - <wars_prototype::main_menu::MainMenu as ggez::event::EventHandler>::draw::hb2c2f643d84dc77c

  22:     0x7ff66ff7affd - winit::platform_impl::platform::event_loop::EventLoop<T>::run_return::{{closure}}::h2abeb118c7ca6fe3

  23:     0x7ff66ffdbe06 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<T>::call_event_handler::{{closure}}::hf5f317343bb018aa

  24:     0x7ff66ffec915 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<T>::register_window::h93e46c905296ea56

  25:     0x7ff66ffebef4 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<T>::move_state_to::h92a0d9d2d74c9932

  26:     0x7ff66ffd8cb5 - winit::platform_impl::platform::event_loop::public_window_callback::{{closure}}::h84c5e107f07f8109

  27:     0x7ff66ffeba18 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<T>::catch_unwind::hd381bb069cc53cc4

  28:     0x7ff6700238c1 - winit::platform_impl::platform::event_loop::public_window_callback::h603be751a752f444

  29:     0x7ffe2bfdd3b6 - DefSubclassProc

  30:     0x7ffe2bfdd26c - DefSubclassProc

  31:     0x7ffe4d38e858 - CallWindowProcW

  32:     0x7ffe4d38e4ee - CallWindowProcW

  33:     0x7ffe1583f190 - glPushClientAttrib

  34:     0x7ffe4d38e858 - CallWindowProcW

  35:     0x7ffe4d38e3dc - DispatchMessageW

  36:     0x7ffe4d3a0bc3 - SendMessageTimeoutW

  37:     0x7ffe4e3b0454 - KiUserCallbackDispatcher

  38:     0x7ffe4c061704 - NtUserDispatchMessage

  39:     0x7ffe4d38e2ea - DispatchMessageW

  40:     0x7ff66ff9b6a4 - winit::platform_impl::platform::event_loop::EventLoop<T>::run::hff9f209cfaeb94c8

  41:     0x7ff66ff4d7a2 - winit::event_loop::EventLoop<T>::run::hc73ad6ee9ebd3d1c

  42:     0x7ff66ff4d3c2 - ggez::event::run::hecda4af9e55f5b56

  43:     0x7ff66ff66a06 - wars_prototype::game::Game::try_command::h9c69a68c091df344

  44:     0x7ff66ff52106 - std::sys_common::backtrace::__rust_begin_short_backtrace::he36f01aa371e1704

  45:     0x7ff66ff4834c - std::rt::lang_start::{{closure}}::h3f3123805b30608b

  46:     0x7ff6701ad463 - core::ops::function::impls::{{impl}}::call_once
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\core\src\ops\function.rs:280

  47:     0x7ff6701ad463 - std::panicking::try::do_call
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:381

  48:     0x7ff6701ad463 - std::panicking::try
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:345

  49:     0x7ff6701ad463 - std::panic::catch_unwind
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panic.rs:396

  50:     0x7ff6701ad463 - std::rt::lang_start_internal
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\rt.rs:51

  51:     0x7ff66ff675c7 - main

  52:     0x7ff6701ce080 - invoke_main
                               at d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:78

  53:     0x7ff6701ce080 - __scrt_common_main_seh
                               at d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288

  54:     0x7ffe4c6d7034 - BaseThreadInitThunk

  55:     0x7ffe4e35d241 - RtlUserThreadStart
```

 - OS: Windows 10
 - Graphics card: NVIDIA GeForce GTX 1050 Ti

**Comment by icefoxen** [comment]
Nice, thank you!  Looks like the panic starts down in `glyph_brush`.  Do you have a stack trace with line numbers for `
0x7ff67001a95f - glyph_brush_draw_cache::DrawCache::cache_queued::h813f93459e4796ca`?

**Comment by junkmail22** [comment]
Here we are:

```thread 'main' panicked at 'attempted to leave type `linked_hash_map::Node<u32, glyph_brush_draw_cache::Row>` uninitialized, which is invalid', /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\core\src\mem\mod.rs:659:9
stack backtrace:
   0:     0x7ff7777f9b7e - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:108
   1:     0x7ff7777f9b7e - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
   2:     0x7ff7777f9b7e - std::sys_common::backtrace::_print_fmt
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:67
   3:     0x7ff7777f9b7e - std::sys_common::backtrace::_print::{{impl}}::fmt
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:46
   4:     0x7ff7778132db - core::fmt::write
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\core\src\fmt\mod.rs:1078
   5:     0x7ff7777f4078 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\io\mod.rs:1518
   6:     0x7ff7777fc9dd - std::sys_common::backtrace::_print
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:49
   7:     0x7ff7777fc9dd - std::sys_common::backtrace::print
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:36
   8:     0x7ff7777fc9dd - std::panicking::default_hook::{{closure}}
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:208
   9:     0x7ff7777fc5b8 - std::panicking::default_hook
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:227
  10:     0x7ff7777fd34f - std::panicking::rust_panic_with_hook
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:593
  11:     0x7ff7777fce23 - std::panicking::begin_panic_handler::{{closure}}
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:497
  12:     0x7ff7777fa4bf - std::sys_common::backtrace::__rust_end_short_backtrace<closure-0,!>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\sys_common\backtrace.rs:141
  13:     0x7ff7777fcda9 - std::panicking::begin_panic_handler
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:495
  14:     0x7ff777810db0 - core::panicking::panic_fmt
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\core\src\panicking.rs:92
  15:     0x7ff777810cfc - core::panicking::panic
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\core\src\panicking.rs:50
  16:     0x7ff7771488a8 - core::mem::uninitialized
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\core\src\mem\mod.rs:659
  17:     0x7ff7771488a8 - linked_hash_map::LinkedHashMap<u32, glyph_brush_draw_cache::Row, core::hash::BuildHasherDefault<rustc_hash::FxHasher>>::ensure_guard_node<u32,glyph_brush_draw_cache::Row,core::hash::BuildHasherDefault<rustc_hash::FxHasher>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\linked-hash-map-0.5.2\src\lib.rs:174
  18:     0x7ff777148bad - linked_hash_map::LinkedHashMap<u32, glyph_brush_draw_cache::Row, core::hash::BuildHasherDefault<rustc_hash::FxHasher>>::insert<u32,glyph_brush_draw_cache::Row,core::hash::BuildHasherDefault<rustc_hash::FxHasher>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\linked-hash-map-0.5.2\src\lib.rs:304
  19:     0x7ff777157b02 - glyph_brush_draw_cache::DrawCache::cache_queued<ab_glyph::font_arc::FontArc,closure-0>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\glyph_brush_draw_cache-0.1.4\src\lib.rs:748
  20:     0x7ff777147b91 - glyph_brush::glyph_brush::GlyphBrush<ggez::graphics::drawparam::DrawParam, glyph_brush::extra::Extra, ab_glyph::font_arc::FontArc, twox_hash::std_support::sixty_four::RandomXxHashBuilder64>::process_queued<ab_glyph::font_arc::FontArc,ggez::graphics::drawp
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\glyph_brush-0.7.2\src\glyph_brush.rs:487
  21:     0x7ff7770c8578 - ggez::graphics::text::draw_queued_text<ggez::graphics::drawparam::DrawParam>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\ggez-0.6.0-rc1\src\graphics\text.rs:542
  22:     0x7ff7770c81d5 - ggez::graphics::text::{{impl}}::draw
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\ggez-0.6.0-rc1\src\graphics\text.rs:418
  23:     0x7ff776eaa56a - ggez::graphics::draw<ggez::graphics::text::Text,ggez::graphics::drawparam::DrawParam>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\ggez-0.6.0-rc1\src\graphics\mod.rs:469
  24:     0x7ff776f5b5c4 - wars_prototype::main_menu::{{impl}}::draw
                               at D:\rust_projects\wars-repo\src\main_menu.rs:61
  25:     0x7ff776eb2601 - ggez::event::run::{{closure}}<wars_prototype::main_menu::MainMenu>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\ggez-0.6.0-rc1\src\event.rs:301
  26:     0x7ff776ef53e8 - winit::platform_impl::platform::event_loop::{{impl}}::run_return::{{closure}}<tuple<>,closure-0>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop.rs:203
  27:     0x7ff7776ce553 - alloc::boxed::{{impl}}::call_mut<tuple<winit::event::Event<tuple<>>, mut winit::event_loop::ControlFlow*>,FnMut<tuple<winit::event::Event<tuple<>>, mut winit::event_loop::ControlFlow*>>,alloc::alloc::Global>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\alloc\src\boxed.rs:1314
  28:     0x7ff77769e8ab - winit::platform_impl::platform::event_loop::runner::{{impl}}::call_event_handler::{{closure}}<tuple<>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop\runner.rs:245
  29:     0x7ff7776afe16 - std::panic::{{impl}}::call_once<tuple<>,closure-0>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panic.rs:322
  30:     0x7ff7776a0aa0 - std::panicking::try::do_call<std::panic::AssertUnwindSafe<closure-0>,tuple<>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panicking.rs:381
  31:     0x7ff7776a64d7 - winit::icon::{{impl}}::clone
  32:     0x7ff7776a08ca - std::panicking::try<tuple<>,std::panic::AssertUnwindSafe<closure-0>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panicking.rs:345
  33:     0x7ff7776b1856 - std::panic::catch_unwind<std::panic::AssertUnwindSafe<closure-0>,tuple<>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panic.rs:396
  34:     0x7ff77769dd39 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<tuple<>>::catch_unwind<tuple<>,tuple<>,closure-0>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop\runner.rs:152
  35:     0x7ff77769e641 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<tuple<>>::call_event_handler<tuple<>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop\runner.rs:239
  36:     0x7ff77769f063 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<tuple<>>::move_state_to<tuple<>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop\runner.rs:341
  37:     0x7ff77769e5c7 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<tuple<>>::main_events_cleared<tuple<>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop\runner.rs:227
  38:     0x7ff7776c8e08 - winit::platform_impl::platform::event_loop::flush_paint_messages<tuple<>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop.rs:676
  39:     0x7ff77711535d - winit::platform_impl::platform::event_loop::public_window_callback::{{closure}}<tuple<>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop.rs:834
  40:     0x7ff77704d2a7 - core::ops::function::FnOnce::call_once<closure-0,tuple<>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\core\src\ops\function.rs:227
  41:     0x7ff776fd0853 - std::panic::{{impl}}::call_once<isize,closure-0>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panic.rs:322
  42:     0x7ff776ff6ade - std::panicking::try::do_call<std::panic::AssertUnwindSafe<closure-0>,isize>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panicking.rs:381
  43:     0x7ff7770105a7 - ggez::timer::TimeContext::tick
  44:     0x7ff776ff67b3 - std::panicking::try<isize,std::panic::AssertUnwindSafe<closure-0>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panicking.rs:345
  45:     0x7ff776fd69de - std::panic::catch_unwind<std::panic::AssertUnwindSafe<closure-0>,isize>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\panic.rs:396
  46:     0x7ff777001093 - winit::platform_impl::platform::event_loop::runner::EventLoopRunner<tuple<>>::catch_unwind<tuple<>,isize,closure-0>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop\runner.rs:152
  47:     0x7ff777114baf - winit::platform_impl::platform::event_loop::public_window_callback<tuple<>>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop.rs:1912
  48:     0x7ffe2bfdd3b6 - DefSubclassProc
  49:     0x7ffe2bfdd26c - DefSubclassProc
  50:     0x7ffe4d38e858 - CallWindowProcW
  51:     0x7ffe4d38e4ee - CallWindowProcW
  52:     0x7ffe3951f190 - glPushClientAttrib
  53:     0x7ffe4d38e858 - CallWindowProcW
  54:     0x7ffe4d38e3dc - DispatchMessageW
  55:     0x7ffe4d3a0bc3 - SendMessageTimeoutW
  56:     0x7ffe4e3b0454 - KiUserCallbackDispatcher
  57:     0x7ffe4c061704 - NtUserDispatchMessage
  58:     0x7ffe4d38e2ea - DispatchMessageW
  59:     0x7ff776ef5097 - winit::platform_impl::platform::event_loop::EventLoop<tuple<>>::run_return<tuple<>,closure-0>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop.rs:218
  60:     0x7ff776ef5446 - winit::platform_impl::platform::event_loop::EventLoop<tuple<>>::run<tuple<>,closure-0>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\platform_impl\windows\event_loop.rs:188
  61:     0x7ff776e975cd - winit::event_loop::EventLoop<tuple<>>::run<tuple<>,closure-0>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.24.0\src\event_loop.rs:154
  62:     0x7ff776eb1aab - ggez::event::run<wars_prototype::main_menu::MainMenu>
                               at C:\Users\junkmail\.cargo\registry\src\github.com-1ecc6299db9ec823\ggez-0.6.0-rc1\src\event.rs:171
  63:     0x7ff776f7d8de - wars_prototype::view::View::run<wars_prototype::main_menu::MainMenu>
                               at D:\rust_projects\wars-repo\src\view.rs:72
  64:     0x7ff776e8974b - wars_prototype::view::View::initial_menu
                               at D:\rust_projects\wars-repo\src\view.rs:79
  65:     0x7ff776f5a321 - wars_prototype::main
                               at D:\rust_projects\wars-repo\src\main.rs:29
  66:     0x7ff776f5fb5b - core::ops::function::FnOnce::call_once<fn(),tuple<>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\core\src\ops\function.rs:227
  67:     0x7ff776ec263b - std::sys_common::backtrace::__rust_begin_short_backtrace<fn(),tuple<>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\sys_common\backtrace.rs:125
  68:     0x7ff776f787a1 - std::rt::lang_start::{{closure}}<tuple<>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\rt.rs:66
  69:     0x7ff7777fd693 - core::ops::function::impls::{{impl}}::call_once
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\core\src\ops\function.rs:280
  70:     0x7ff7777fd693 - std::panicking::try::do_call
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:381
  71:     0x7ff7777fd693 - std::panicking::try
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panicking.rs:345
  72:     0x7ff7777fd693 - std::panic::catch_unwind
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\panic.rs:396
  73:     0x7ff7777fd693 - std::rt::lang_start_internal
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\/library\std\src\rt.rs:51
  74:     0x7ff776f78773 - std::rt::lang_start<tuple<>>
                               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca\library\std\src\rt.rs:65
  75:     0x7ff776f5a350 - main
  76:     0x7ff777836d40 - invoke_main
                               at d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:78
  77:     0x7ff777836d40 - __scrt_common_main_seh
                               at d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288
  78:     0x7ffe4c6d7034 - BaseThreadInitThunk
  79:     0x7ffe4e35d241 - RtlUserThreadStart```

**Comment by PSteinhaus** [comment]
If you still have your old build could you please check your cargo lockfile to see whether you're using ```0.5.4``` of linked-hash-map?
If not could you please update us on whether this issue still persists for you?

**Comment by simplesmiler** [comment]
Had the same exact problem, updating linked-hash-map to 0.5.4 fixed it.

**Comment by PSteinhaus** [comment]
Haha! Thank you! That's exactly what we needed to know!
Then I think we can safely call this fixed, since ggez now uses 0.5.4.
